schema/places/place.yaml

```diff
@@ -41,6 +41,23 @@ properties:
             items:
               "$ref": "./defs.yaml#/$defs/typeDefinitions/category"
             uniqueItems: true
+      basic_level_category:
+        description: |
+          The Basic-level categorization is a cognitive science model that is relevant
+          for taxonomy and ontology development that shows the most broadest and general
+          category name that is most often found in the middle of a general-to-specific
+          hierarchy, with generalization proceeding upward and specialization
+          proceeding downward. The full list of basic level categories is available at:(todo)
+        type: object
+        required: [basic]
+        properties:
+          basic:
```

Comment:
This is introducing multiple levels in the hierarchy. You're going to get something like this in JSON:

```json
{
  "properties": {
    "basic_level_category": {
      "basic": [
        "coffee_shop",
        "pet_shop"
      ]
    }
  }
}
```

And in tabular format you're going to end up with a STRUCT column that contains an ARRAY of STRING.

I'm almost certain that's at least one level of nesting more than you are intending.

---

Questions:

1. Does a Place have one basic category or many?
2. If it has many, are you ending to have structured data for each category, or just a single category ID?